CONTRIBUTING.md

```diff
@@ -1,113 +1,29 @@
-					events.png
-					...
-				/sample
-					(sample code)
-			/bootstrap
-				/_static
-					about-page.png
-					...
-			angular.rst
-			bootstrap.rst
-
-**Note:** Sphinx will automatically fix duplicate image names, such as the about-page.png files shown above. There is no need to try to ensure uniqueness of static files beyond an individual article.
-
-Author information should be placed in the _authors folder following the example of steve-smith.rst. Place photos in the photos folder - size them to be no more than 125px wide or tall.
-
-## Process for Contributing ##
-
-**Step 1:** Open an Issue describing the article you wish to write and how it relates to existing content. Get approval to write your article.
-
-**Step 2:** Fork the `/aspnet/docs` repo.
-
-**Step 3:** Create a `branch` for your article.
-
-**Step 4:** Write your article, placing the article in its own folder and any needed images in a _static folder located in the same folder as the article. Be sure to follow the [ASP.NET Docs Style Guide](http://docs.asp.net/en/latest/contribute/style-guide.html). If you have code samples, place them in a folder within the `/samples/` folder.
-
-**Step 5:** Submit a Pull Request from your branch to `aspnet/docs/master`.
-
-**Step 6:** Discuss the Pull Request with the ASP.NET team; make any requested updates to your branch. When they are ready to accept the PR, they will add a :shipit: (`:shipit:`) comment.
-
-**Step 7:** The last step before your Pull Request is accepted is to [squash all commits](http://stackoverflow.com/questions/14534397/squash-all-my-commits-into-one-for-github-pull-request) into a single commit message. Do this in your branch, using the `rebase` git command. For example, if you want to squash the last 4 commits into a single commit, you would use:
-
-	git rebase -i HEAD~4
-
-The `-i` option stands for "interactive" and should open a text editor showing the last N commits, preceded with "pick ".  Change all but the first instance of "pick " to "squash " and save the file and exit the editor. A more detailed answer is [available here](http://stackoverflow.com/a/6934882).
-
-## Common Pitfalls ##
+The command above starts the preview on localhost:8080 where you can view the updated content in your browser.
 
-Below are some common pitfalls you should try to avoid:
+Notes:
 
-- Don't forget to submit an issue before starting work on an article
-- Don't forget to create a separate branch before working on your article
-- Don't update or `merge` your branch after you submit your pull request
-- Don't forget to squash your commits once your pull request is ready to be accepted
-- If updating code samples in `/samples/`, be sure any line number references in your article remain correct
+* Currently DocFX requires the .NET Framework on Windows or Mono (for Linux or macOS)
+* The local preview currently doesn't contain themes or render table correctly; the view will differ slightly from the published site
```

Comment:
~~Currently~~
Also doesn't do some MS Docs extensions
"correctly" doesn't fit , could omit mentioning tables as that is covered adequately by "themes"